src/targets/gpu/pack_int8_args.cpp

```diff
@@ -16,24 +22,42 @@ void pack_int8_args::apply(module& p) const
     {
         if(ins->name() == "gpu::quant_gemm")
         {
+            auto&& op = any_cast<rocblas_gemm<op::quant_dot>>(ins->get_operator());
```

Comment:
Is there a reason this doesn't query the flag here?

Reply:
What does it mean?

Reply:
I think he is asking to use the rocBLAS query API here instead of lowering.

Reply:
Dont use `any_cast`, instead read the operator with `to_value`.

Reply:
fixed